src/main/java/net/hypixel/nerdbot/command/ItemGenCommand.java

```diff
@@ -37,7 +37,8 @@ public void askForInfo(
             @AppOption(description = "The rarity of the item", autocomplete = "rarities") String rarity,
             @AppOption(description = "The description of the item") String description,
             @Optional @AppOption(description = "The type of the item") String type,
-            @Optional @AppOption(description = "If you will handle line breaks at the end of the item's description") Boolean handleLineBreaks
+            @Optional @AppOption(description = "If you will handle line breaks at the end of the item's description") Boolean handleLineBreaks,
+            @Optional @AppOption(description = "If you want the image to be semitransparent") Boolean transparent

```

Comment:
```suggestion
            @Optional @AppOption(description = "If you want the image to be semi-transparent") Boolean transparent
```

Reply:
![image](https://user-images.githubusercontent.com/60010287/219204241-8022117d-0efc-42f4-bbfd-f0ba8e079fa4.png)

Reply:
Google is lying to me